JetAnalyzers/src/fitCrystalBall.cc

```diff
@@ -0,0 +1,1043 @@
+#include "JetMETAnalysis/JetAnalyzers/interface/crystalBall.h"
+#include "JetMETAnalysis/JetAnalyzers/interface/crystalBallRes.h"
+
+#include <TROOT.h>
+#include <TSystem.h>
+#include <TFile.h>
+#include <TKey.h>
+#include <TH1F.h>
+#include <TF1.h>
+#include <TTree.h>
+#include "TVirtualFitter.h"
+#include "TMath.h"
+#include "TSpectrum.h"
+#include "TRandom3.h"
+#include "TCanvas.h"
+
+#include "TMath.h"
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <math.h>
+#include <map>
+#include <algorithm>
```

Comment:
Please move all of the headers except for the first one to the fitCrystalBall.hh file. Not only will this keep the style consistent, but it will make sure that all of the headers are within header guards.